plugins/services/src/js/components/modals/EditServiceModal.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+
+import EditFrameworkConfiguration
+  from "#PLUGINS/services/src/js/pages/EditFrameworkConfiguration";
+import CreateServiceModal
+  from "#PLUGINS/services/src/js/components/modals/CreateServiceModal";
+import Framework from "#PLUGINS/services/src/js/structs/Framework";
+import DCOSStore from "#SRC/js/stores/DCOSStore";
+
+class EditServiceModal extends Component {
+  render() {
+    const { params } = this.props;
+
+    const serviceID = decodeURIComponent(params.id || "/");
+    const service = DCOSStore.serviceTree.findItemById(serviceID);
+
+    let child = null;
+    if (service instanceof Framework) {
+      child = <EditFrameworkConfiguration {...this.props} />;
```

Comment:
I'd simply return Instead of assigning the `child` to improve the readability.

```js
class EditServiceModal extends Component {
  render() {
    const { id  = "/" } = this.props.params;
    const service = DCOSStore.serviceTree.findItemById(decodeURIComponent(id));

    if (service instanceof Framework) {
      return (<FrameworkConfiguration {...this.props} />);
    } 
    
   return (<CreateServiceModal {...this.props} />);
  }
}
```